SunSpec Battery: Broken Test

### Describe the bug

The sunspec-inverter/battery tests are failing when executing locally.

### Steps to reproduce

1. make test

### Configuration details

```yaml
N/A
```


### Log details

```text
--- FAIL: TestTemplates (0.00s)
    --- FAIL: TestTemplates/sunspec-inverter/battery (0.00s)
        render_testing.go:88: map[baudrate:9600 capacity:50 comset:8N1 device:/dev/ttyUSB0 host:localhost id:1 modbus: port:502 tcpip:true template:sunspec-inverter usage:battery]
        render_testing.go:88: cannot create meter type 'template': cannot create meter type 'custom': power: missing configuration
FAIL
FAIL	github.com/evcc-io/evcc/meter	31.895s
```


### What type of operating system are you running?

Linux

### Version

master @ 3bf57f870ee66995c9a01784d93d9ac7e5ce7e02

--- a/templates/definition/meter/sunspec-inverter.yaml
+++ b/templates/definition/meter/sunspec-inverter.yaml
@@ -102,6 +102,16 @@ render: |
   {{- end }}
   {{- if eq .usage "battery" }}
   type: custom
+  power:
+    source: sunspec
+    {{- include "modbus" . | indent 2 }}
+    value:
+      - 101:W
+      - 103:W
+      - 111:W
+      - 113:W
+      - 102:W
+      - 112:W
   soc:
     source: sunspec
     {{- include "modbus" . | indent 2 }}

